tests/phpunit/Console/E2ETest.php

```diff
@@ -69,10 +69,14 @@
 use Symfony\Component\Process\Process;
 
 /**
- * @group e2e
```

Comment:
What's the reason for removing this group?

I find it useful to run `--exclude-group=e2e` and run e2e tests separately by `tests/e2e bin/infection` or `tests/e2e build/infection.phar`.

Now, this is impossible.

Reply:
No problem, I got it back.